index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO: SETUP AND MOBILE VERSION SKELETON</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div id="toolbar">
+                <nav class="left">
+                    <img src="Time.png" alt="">
+                </nav>
+                <nav class="right">
+                    <img src="Cellular.png" alt="Cellular">
+                    <img src="Wifi.png" alt="Wifi">
+                    <img src="Battery.png" alt="battery">
+                </nav>
+                <nav>
+                    <div class="toolbar1">
+                        <img src="Logo.png" alt="Logo">
+                    </div>
+                    <div class="toolbar2">
+                        <img src="Union.png" alt="Menu">
+                    </div>
+                </nav> 
+            </div>
+        </header>
+        <main class="name">
+            <p>I'm Kelvin Glad to see you!</p>
```

Comment:
- [ ] It will be nice to make this line an `h1` instead of `p` tag. It will be clear because this is the main title and not a paragraph.